README.md

```diff
@@ -48,60 +82,58 @@ Copy `sample.env` to `.env` and make the following changes within the `.env`:
 
 -   Set `SEND_PAYOUT_THRESHOLD` to a minimum threshold which payout amounts must exceed to be sent. Default is 0, and payout transaction amount must _exceed_ this number to be sent. Also specified in Mina!
 
 -   Set `SEND_PRIVATE_KEY` to the sender private key
     The private key value can be retrieved from a pk file by running the mina advanced dump-keypair command, e.g.
 
-    ```
-    mina advanced dump-keypair --privkey-path keys/my-payout-wallet
-    ```
+```
+mina advanced dump-keypair --privkey-path keys/my-payout-wallet
+```
 
 -   Set `SEND_PUBLIC_KEY` to the sender public key. It can also be blank if generating ephemeral keys.
 
 -   Set `MIN_CONFIRMATIONS` to whatever number of confirmed blocks you require before paying out. Default to 290 or "k" to use the assumed network finality.
 
-    The process will include blocks at a height up to the **lower of** `MAX_HEIGHT` and the current tip minus `MIN_CONFIRMATIONS`.
+The process will include blocks at a height up to the **lower of** `MAX_HEIGHT` and the current tip minus `MIN_CONFIRMATIONS`.
 
-    To clarify - `MAX_HEIGHT` only applies _below_ the minimum confirmation window. (i.e. Given a current block height of 1,500; `MAX_HEIGHT` of 5,000; and `MIN_CONFIRMATIONS` of 290, the process will consider blocks up to height 1210 (1500-290). If `MAX_HEIGHT` were set to 1,000, then the process would consider blocks up to height 1000.)
+To clarify - `MAX_HEIGHT` only applies _below_ the minimum confirmation window. (i.e. Given a current block height of 1,500; `MAX_HEIGHT` of 5,000; and `MIN_CONFIRMATIONS` of 290, the process will consider blocks up to height 1210 (1500-290). If `MAX_HEIGHT` were set to 1,000, then the process would consider blocks up to height 1000.)
 
 -   Populate `DATABASE_URL` with the connection string for your archive node Postgresql instance. This will typically look something like:
 
-    ```
-    DATABASE_URL=postgresql://USER:PASSWORD@HOST:PORT/DATABASENAME
-    ```
+```
+DATABASE_URL=postgresql://USER:PASSWORD@HOST:PORT/DATABASENAME
+```
 
 -   Set `SEND_PAYMENT_GRAPHQL_ENDPOINT` to the url of a graphql server that can send transactions. (e.g. http://127.0.0.1:3085/graphql ) This is required to transmit payout transactions; payouts will be broadcast via this endpoint.
 
-    -   Note @vanphandinh / beaconchain#8571 has provided a graphql proxy https://github.com/vanphandinh/mina-graphql-proxy which can be used to transmit payments
-    -   beaconchain is providing a public endpoint here: https://minagraph.com/graphql (provided with permission, and _do your own research_)
+-   Note @vanphandinh / beaconchain#8571 has provided a graphql proxy https://github.com/vanphandinh/mina-graphql-proxy which can be used to transmit payments
+-   beaconchain is providing a public endpoint here: https://minagraph.com/graphql (provided with permission, and _do your own research_)
```

Comment:
This is now deprecated. I'll have https://proxy.minaexplorer.com/graphql running till the hardfork, as per the archive service.